docs/app/pages/components/components-sections/badge/badge/badge.component.html

```diff
@@ -0,0 +1,69 @@
+<p>Demo here</p>
+
+<hr>
+
+<p>
+    The <code>ux-badge</code> directive can be used to easily attached a badge to any DOM element to be used as a label/tag, or to show (x) number of new items relating to it etc.
+</p>
+
+<p>The appearance and behavior of the directive can be configured using the following attributes:</p>
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="badgeBackgroundColor" type="string">
```

Comment:
Can I also suggest adding a `badgeHidden` property. There could be occasions whenever we want to conditionally hide the badge